src/System.Management.Automation/engine/remoting/commands/RemoveJob.cs

```diff
@@ -46,10 +46,10 @@ public class JobCmdletBase : PSRemotingCmdlet
         /// Find the jobs in repository which match matching the specified names
         /// </summary>
         /// <param name="writeobject">if true, method writes the object instead of returning it
-        /// in list (an empty list is returned). </param>
-        /// <param name="writeErrorOnNoMatch">write error if no match is found</param>
-        /// <param name="checkIfJobCanBeRemoved">check if this job can be removed</param>
-        /// <param name="recurse">recurse and check in child jobs</param>
+        /// in list (an empty list is returned). .</param>
```

Comment:
Double period.  Just remove whitespace.  Perhaps just globally search for ".  .</param>" and replace with ".</param>"

Reply:
I reverted the changes in the file.